benchmarks/nemo/nemolite2d/manual_versions/psykal_c_opencl/time_step.c

```diff
@@ -232,8 +203,8 @@ void c_invoke_time_step(
             /* The Intel/Altera OpenCL SDK is only version 1.0 */
             /* NVIDIA only support OpenCL 1.2 so we get a seg. fault if we attempt
             to call the ...WithProperties version of this routine */
-            command_queue[ji] = clCreateCommandQueue(
-                    context, device, queue_properties, &ret);
+            command_queue[ji] = clCreateCommandQueueWithProperties(
```

Comment:
I changed it because I was always getting a warning that `clCreateCommandQueue` is being deprecated. It does work in Glados with Cuda10 but I am not sure about the Altera SDK, so I still left the comment that it may not work with Altera.